modules/nw-ovn-kubernetes-rollback-live.adoc

```diff
@@ -71,9 +71,18 @@ The command prints the following information every second:
 * The conditions on the status of the `network.config.openshift.io/cluster` object, reporting the progress of the migration.
 * The status of different nodes with respect to the `machine-config-operator` resource, including whether they are upgrading or have been upgraded, as well as their current and desired configurations.
 
-. After the rollback procedure has completed, enter the following command to remove the `network.openshift.io/network-type-migration=` annotation from the `network.config` custom resource:
+. Complete the following steps only if the migration succeeds and your cluster is in a good state:
++
+.. Remove the `network.openshift.io/network-type-migration=` annotation from the `network.config` custom resource by entering the following command:
 +
 [source,terminal]
 ----
 $ oc annotate network.config cluster network.openshift.io/network-type-migration-
+----
++
+..Remove the OVN-Kubernetes network provider namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc delete namespace openshift-ovn-kubernetes
```

Comment:
My only question is if it is ok to delete the `openshift-ovn-kubernetes` namespace. I'm not an engr though so 🤔 .